utils/atheepmgr/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2021 Sergey Ryazanov <ryazanov.s.a@gmail.com>
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=atheepmgr
+PKG_VERSION:=2.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rsa9000/$(PKG_NAME).git
+PKG_MIRROR_HASH:=2c0c9705b784f736d1fb32777849d1cc7cefeee0137512f0624f5045ebf6fef6
```

Comment:
Will switch to .tar.xz. Nice catch, thank you!

Reply:
As for version, archives and for build directory overriding, I intentionally configured sources info in such way:
* to avoid manual PKG_BUILD_DIR override to keep package as clean as possible;
* to workaround github habit to prepend repository name everywhere.

So I prefer to keep source code related variables in there current state. Or I should you use some specific API to download from github?

Reply:
There's a possibility to use in your case git and checkout to specific commit hash, but that's does not seems like a good approach as we should stick to releases or tags.  There is nothing wrong to overwride `PKG_BUILD_DIR`. If you want to use still use git, I will come with some solution later.

Reply:
Is it worth to reinvent the quite working downloading approach? I am still do not see any reason to doing so.